app/models/in_person_enrollment.rb

```diff
@@ -37,6 +39,10 @@ def usps_unique_id
     user.uuid.delete('-').slice(0, 18)
   end
 
+  def self.generate_unique_id
```

Comment:
Will you add a comment to explain that we're generating a random 18-digit string? It's a little mysterious that we pass in `9` until you understand that hex returns a string of length n*2

Reply:
Done